Consts.py

```diff
@@ -29,3 +29,6 @@
 
 # line length for dynamic smoot path
 length_draw = 1
+
+# degree frequency at which to generate lidar data
+degree_freq = 1
```

Comment:
Maybe we can improve the efficiency by scanning every few degrees but still catch all the objects?

Reply:
I did try that, but the drawing looks bad if degree_freq is any larger than 1 :(